src/ubx.cpp

```diff
@@ -2039,6 +2188,47 @@ GPSDriverUBX::payloadRxDone()
 			ret = 1;
 		}
 
+		{
+			sensor_gnss_relative_s gps_rel{};
```

Comment:
This was an extra topic added upstream by dagar.  Would be useful data to have, if the Ark had an SD card.

Reply:
If we really wanted to, we could serialize this on the CAN bus with a custom UAVCAN topic.

Reply:
Let's aim to add this. It's not being consumed by EKF2 but would be great to have in logs and update ecl_ekf to use this rather than heading angle for updates.

Reply:
Added!  (It's in the PX4 PR now)